backend/server.js

```diff
@@ -42,9 +43,12 @@ const io = socketio(server)
 app.set('io', io)
 
 
+
 app.use(express.static(path.join(__dirname, '/../client/build')));
 // Direct other requests to the auth router
-app.use('/', authRouter)
+app.use('/', authRouter);
+
+routes.loginRoutes(app);
```

Comment:
What does it mean?

Reply:
server.js is delegating the routing of the login operation to the routes.loginRoutes